### ISSUE TITLE
 Best accuracy: 39.79 even on CIFAR-10

### ISSUE THREAD

**Issue by CodEros9231** [CodEros9231/netadapt#1]
I have run the following command, and it has given me an accuracy of 39.79% even on CIFAR-10.. Not sure what's the issue. I have run it for 92 epochs so far : 
ek3296@ADUAED19221LPLX:~/netadapt$ python train.py data/ --dir models/aspp/aspp_model.tar --arch aspp --batch-size 16 --resume models/aspp/aspp_model.tar
AtrousSpatialPyramidPooling(
  (aspp1): ASPPModule(
    (conv): Conv2d(3, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
    (bn): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu): ReLU(inplace=True)
  )
  (aspp2): ASPPModule(
    (conv): Conv2d(3, 256, kernel_size=(3, 3), stride=(1, 1), padding=(6, 6), dilation=(6, 6), bias=False)
    (bn): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu): ReLU(inplace=True)
  )
  (aspp3): ASPPModule(
    (conv): Conv2d(3, 256, kernel_size=(3, 3), stride=(1, 1), padding=(12, 12), dilation=(12, 12), bias=False)
    (bn): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu): ReLU(inplace=True)
  )
  (aspp4): ASPPModule(
    (conv): Conv2d(3, 256, kernel_size=(3, 3), stride=(1, 1), padding=(18, 18), dilation=(18, 18), bias=False)
    (bn): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu): ReLU(inplace=True)
  )
  (concat_projection): Sequential(
    (0): Conv2d(1024, 256, kernel_size=(1, 1), stride=(1, 1), bias=False)
    (1): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (2): ReLU(inplace=True)
    (3): Dropout(p=0.5, inplace=False)
  )
  (fc): Linear(in_features=256, out_features=10, bias=True)
)
torch.Size([1, 10])
Namespace(data='data/', workers=4, epochs=150, start_epoch=0, arch='aspp', batch_size=16, lr=0.1, momentum=0.9, weight_decay=0.0005, resume='models/aspp/aspp_model.tar', save_dir='models/aspp/aspp_model.tar', no_cuda=False)
Files already downloaded and verified
Files already downloaded and verified
Loading checkpoint 'models/aspp/aspp_model.tar'
Resumed from checkpoint: models/aspp/aspp_model.tar, starting at epoch 0, best accuracy so far: 30.84
Epoch [1/150]
===================================================================
Percent: [##########] 99.97% (3124/3125) ESA:     0.00s, loss: 0.29, acc: 18.75%
Finish epoch 1: time =   331.53s, loss = 0.27, acc = 33.61%
